src/libraries/System.Threading.Tasks.Dataflow/src/Blocks/TransformManyBlock.cs

```diff
@@ -64,9 +64,12 @@ public sealed class TransformManyBlock<TInput, TOutput> : IPropagatorBlock<TInpu
         /// <param name="dataflowBlockOptions">The options with which to configure this <see cref="TransformManyBlock{TInput,TOutput}"/>.</param>
         /// <exception cref="System.ArgumentNullException">The <paramref name="transform"/> is null (Nothing in Visual Basic).</exception>
         /// <exception cref="System.ArgumentNullException">The <paramref name="dataflowBlockOptions"/> is null (Nothing in Visual Basic).</exception>
-        public TransformManyBlock(Func<TInput, IEnumerable<TOutput>> transform, ExecutionDataflowBlockOptions dataflowBlockOptions) :
-            this(transform, null, dataflowBlockOptions)
-        { }
+        public TransformManyBlock(Func<TInput, IEnumerable<TOutput>> transform, ExecutionDataflowBlockOptions dataflowBlockOptions)
+        {
+            // Validate arguments.
+            if (transform == null) throw new ArgumentNullException(nameof(transform));
```

Comment:
Nit: Add Braces

*Personally, I also prefer the style in such cases. But the runtime coding style specifies that you should use braces.*
https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md (18.)